spec/rubocop/cop/graphql/ordered_fields_spec.rb

```diff
@@ -107,6 +108,56 @@ class UserType < BaseType
     end
   end
 
+  context "when fields are not consecutive" do
+    context "when there are blocks" do
+      it "registers an offense" do
+        expect_offense(<<~RUBY)
+          class UserType < BaseType
+            field :phone, String, null: true do
+              argument :something, String, required: false
+            end
+            field :name, String, null: true do
+            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Fields should be sorted in an alphabetical order within their section. Field `name` should appear before `phone`.
+            end
+          end
+        RUBY
+
+        expect_correction(<<~RUBY)
+          class UserType < BaseType
+            field :name, String, null: true do
+            end
+            field :phone, String, null: true do
+              argument :something, String, required: false
+            end
+          end
+        RUBY
+      end
+    end
+
+    context "when there are no blocks" do
+      it "registers an offense" do
+        expect_offense(<<~RUBY)
+          class UserType < BaseType
+            field :phone, String, null: true
+
+
+            field :name, String, null: true
+            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Fields should be sorted in an alphabetical order within their section. Field `name` should appear before `phone`.
+          end
+        RUBY
+
+        expect_correction(<<~RUBY)
+          class UserType < BaseType
+            field :name, String, null: true
+            field :phone, String, null: true
+
+
+          end
```

Comment:
In particular, this auto correct doesn't look so great. I think it's because we use the swap with line ranges.

Reply:
that's fine, we have another cop that leaves spaces